livestream/docker-compose.yml

```diff
@@ -1,62 +1,62 @@
-version: "3.8"
+version: '3.8'
```

Comment:
undo changes here please :)

Reply:
oh lmao no idea how this got here.  Undoing!

Reply:
pro-tip if you're unaware, when you don't have a specific commit to revert, you can bring a file back to its state on master via:

```
git checkout master livestream/docker-compose.yml
```

as long as master is upto date, this will effectively revert all changes in the file!